app/src/main/java/com/mybus/adapter/BusResultViewAdapter.java

```diff
@@ -47,23 +47,20 @@ public void onBindViewHolder(BusResultViewHolder holder, int pos) {
         BusRouteResult routeResult = mDataset.get(pos);
         List<BusRoute> routes = routeResult.getBusRoutes();
         if (routes != null && !routes.isEmpty()) {
+            BusRoute firstBusRoute = routes.get(0);
+            holder.mStartAddress.setText(firstBusRoute.getStartBusStopStreetName() + " " + firstBusRoute.getStartBusStopStreetNumber());
```

Comment:
You are getting this warning:
Do not concatenate text displayed with setText. Use resource string with placeholders. less... (Ctrl+F1)
When calling TextView#setText:
*Never call Number#toString() to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using String#format with proper format specifications (%d or %f) instead.
*Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
*Do not build messages by concatenating text chunks. Such messages can not be properly translated.